newIDE/app/src/MainFrame/Preferences/PreferencesDialog.js

```diff
@@ -261,6 +262,21 @@ const PreferencesDialog = ({ i18n, onClose }: Props) => {
               }
             />
           </Line>
+          <Line>
+            <Toggle
+              onToggle={(e, check) =>
+                setEventsSheetCancelInputParameterPopover(check)
+              }
+              toggled={values.eventsSheetCancelInputParameterPopover}
+              labelPosition="right"
+              label={
+                <Trans>
+                  Use Escape for cancel changes on inline parameter, CTRL+ENTER
```

Comment:
This describes what it does if we activate this, but if we deactivate this what happens?
A Toggle might not be the proper thing to use here.

Reply:
Should also be "... **to** cancle changes ..." I think.

Reply:
Oops, added a new typo. ^^
It is "cancel" of course.

Reply:
Sorry, this review on this line is outdated.
The current line is : 
`Inline popover parameter behavior when escaping` 
I've pinged you on the correct one.